core/src/sumeragi/main_loop.rs

```diff
@@ -936,7 +976,8 @@ fn reset_state(
         *voting_block = None;
         voting_signatures.clear();
         *last_view_change_time = Instant::now();
-        *view_change_time = pipeline_time;
+        *view_change_time =
+            pipeline_time.saturating_mul((view_change_index + 1).try_into().unwrap_or(u32::MAX));
```

Comment:
Why increase the transaction complexity allowance based on `pipeline_time` which includes `block_time`, when block creation seems independent of transaction complexity?